### DISCUSSION TITLE
How to find which tests are failing?

### DISCUSSION THREAD

**Discussion by hasnainzeenwala** [pytest-dev/pytest#10036] [Q&A] [answered]
I'm running all the tests after making my changes and certain tests are failing. The problem is I get information in this format:
test_doctest.py ..........................................................................x........................................................s.                                    [100%]

How can I see what particular test is failing? I'm getting the result in the same format even if I run the file individually using pytest.

**Comment by RonnyPfannschmidt** [comment, marked as answer]
There is no failure, only a expected one and a skip

With `pytest -rxs` you should see the details

**Reply by hasnainzeenwala** [reply]
Got it! Thanks a lot.
